grails-app/domain/org/pih/warehouse/inventory/InventoryItem.groovy

```diff
@@ -40,6 +40,7 @@ class InventoryItem implements Serializable {
     Product product                        // Product that we're tracking
     String lotNumber                        // Lot information for a product
     Date expirationDate
+    Boolean recalled = Boolean.FALSE
```

Comment:
Sorry I should have caught this during the tech huddle. Can you turn this into an enum?

LotStatusCode
APPROVED
RECALLED
ON_HOLD
QUARANTINED
EXPIRED
RESERVED
DAMAGED

Reply:
Sure

Reply:
@jmiranda done